sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -500,6 +500,7 @@ impl InputSelection {
                             foundry_output,
                             input_native_tokens_builder.deref(),
                             output_native_tokens_builder.deref(),
+                            &Default::default(),
```

Comment:
This is currently failing, we need to set the flag is something is being burned I guess

Reply:
I've tried something